### PR TITLE
Restructuring of README

### PR DESCRIPTION
**Pull request by drpatelh**
Thought it would be nice to give this site the same look and feel as `nf-core/configs`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,58 +1,81 @@
...
+### Docker
 
-We provide a Docker Image on [DockerHub](https://cloud.docker.com/u/nfcore/repository/docker/nfcore/logos) that you may use to create logos for your pipeline easily.
+We provide a Docker Image on [DockerHub](https://cloud.docker.com/u/nfcore/repository/docker/nfcore/logos) that you may use to create logos for your pipeline easily:
 
 ```bash
-docker run -v /path/on/host:/data -it nfcore/logos bash
-/make_logo.sh NewPipeline
-mv NewPipeline* /data
+docker run -v `pwd`:`pwd` -w `pwd` nfcore/logos NewPipeline
```
Comment (drpatelh) [inline README.md]: @apeltzer Ive add this in as suggested by @ewels but I have no idea how to implement `ENTRYPOINT` in the `Dockerfile`. Can you send me a file/code-block with the changes, and ill push it.

File: README.md [hunk 2]
```diff
@@ -1,58 +1,81 @@
...
 ```
 
-You will then find your logos in PNG and SVG format in `/path/on/host/`.
+You will find your logos in `png` and `svg` format in `/path/on/host/`.
```
Comment (drpatelh) [inline README.md]: Will `/path/on/host/` have to change here?

File: README.md (hunk 1 above)
Comment (ewels) [reply README.md]: see eg. [here](https://github.com/ewels/MultiQC/blob/39d83c2e5991f8911e7f522987a1c8f5851c9c5b/Dockerfile#L22) - basically exactly as in my [comment](https://github.com/nf-core/logos/pull/7#discussion_r282467198).. 😀

Comment (drpatelh) [reply README.md]: BOOOOM! Like now?

Comment (drpatelh) [reply README.md]: PS: Im registering for ISMB. I dont have time to learn `Docker` right now.

**Comment by apeltzer** [comment]
I think we might have to rewrite the bash script a little so that it can save the output at a  different location instead of writing to the same folder the template SVG is already stored at (which creates trouble in read-only environments, e.g. inside Singularity Containers). Having the entry point as Phil suggested is also a good idea and could simply point the script to `$PWD` the current directory the users are in.

File: README.md (hunk 1 above)
Comment (ewels) [reply README.md]: almost - I don’t think you need the `CMD` though. That’ll make a logo with the title `.` if a name is not specified...

**Pull request merged**